single_tech_samples/databricks/sample4_ci_cd/pyspark/devops/cd-pipeline.yml

```diff
@@ -0,0 +1,93 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger: none
+
+pr: none
+
+resources:
+  pipelines:
+  - pipeline: ci
+    source: mdw-dbx-pyspark-ci-pipeline
+
+stages:
+- stage: IT
+  displayName: Integration Test
+  variables:
+  - group: mdwdo-dbx-pyspark-test
+  - group: mdwdo-dbx-release-dev
+  jobs:
+  - job: ITJob
+    displayName: Integration Test with dbx CLI
+    pool:
+      vmImage: ubuntu-latest
+    steps:
+    - task: AzureKeyVault@1
```

Comment:
We should document how to set this service connection up. What secrets should be in KeyVault?
Also, you are referencing two variable groups, so I'm unsure where you expect the user to set the configuration/variables...

Reply:
Service connection and KeyVaults contents have been added to **[Prerequisites](https://github.com/Azure-Samples/modern-data-warehouse-dataops/tree/single-tech/databricks-ops-thurstonchen/single_tech_samples/databricks/sample4_ci_cd/pyspark#prerequisites)** section, thanks for inputs from @meghapatilcode! :-)